spec/requests/api/v1/users/login_spec.rb

```diff
@@ -29,8 +28,8 @@
     end
   end
 
-  context 'Incorrectly login the user' do
-    context 'Incorrect credentials' do
+  context 'when fails' do
```

Comment:
also could replace `context 'Correctly login the user' do` at line 8 for `context 'when success` and same for the other specs (I understand that are for the last PR, but could be a nice improvement).
The idea behind that is to properly split scenarios, and use a more friendly nomenclature.